src/instruments/src/SD/Pages/Fctl/Fctl.tsx

```diff
@@ -1,738 +1,390 @@
+    const d1 = `M0 0 l${
+        side === 'left' ? '-' : ''}8 0 l0 -20 l${
+        side === 'right' ? '-' : ''}8 0 l0 120 l${side === 'left' ? '-' : ''}8 0 l0 -10 l${side === 'right' ? '-' : ''}8 0`;
+    const d2 = `M0 36 l${side === 'left' ? '-' : ''}7 0`;
+    const d3 = `M0 41 l${side === 'left' ? '-' : ''}7 0`;
+    const d4 = `M0 46 l${side === 'left' ? '-' : ''}8 0 l0 6 l${side === 'right' ? '-' : ''}8 0`;
+
+    return (
+        <SvgGroup x={x} y={y}>
+            <path className="MainShape" d={d1} />
+            <path className="MainShape" d={d2} />
+            <path className="MainShape" d={d3} />
+            <path className="MainShape" d={d4} />
+        </SvgGroup>
+    );
+};
+
+const Elevator = ({ x, y, side, leftHydraulicSystem, rightHydraulicSystem }: ComponentPositionProps & ComponentSidePositionProps & HydraulicSystemPairProps) => {
+    const textPositionX = side === 'left' ? -42 : 42;
+    const textLetter = side === 'left' ? 'L' : 'R';
+
+    const [elevatorDeflection] = useSimVar('ELEVATOR DEFLECTION PCT', 'percent over 100', 50);
+    const elevatorDeflectPctNormalized = elevatorDeflection * (elevatorDeflection > 0 ? 70 : 52);
+    const cursorPath = `M${side === 'left' ? 1 : -1},${70 - elevatorDeflectPctNormalized} l${side === 'right' ? '-' : ''}15,-7 l0,14Z`;
+
+    const hydraulics = useContext(HydraulicContext);
+
+    return (
+        <SvgGroup x={x} y={y}>
+            <Note x={textPositionX} y={0}>{textLetter}</Note>
+            <Note x={textPositionX} y={22}>ELEV</Note>
+
             <path
-                id={`arrow${index}_${leftorright}`}
-                visibility={showspoiler ? 'visible' : 'hidden'}
-                className="GreenShape"
-                d={`M ${leftorright === 'left' ? x + 8 : x - 8} ${y} l 0 -22 l -6 0 l 6 -12 l 6 12 l -6 0`}
+                id={`${side}ElevatorCursor`}
+                className={hydraulics[leftHydraulicSystem].available || hydraulics[rightHydraulicSystem].available ? 'GreenShape' : 'WarningShape'}
+                d={cursorPath}
             />
-            <text
-                id={`num${index}_${leftorright}`}
-                visibility={hydraulicsAvailable ? 'hidden' : 'visible'}
-                className="Warning Medium"
-                x={`${leftorright === 'left' ? x + 8 : x - 8}`}
-                y={`${yw}`}
-                textAnchor="middle"
-                alignmentBaseline="central"
-            >
-                {index}
-            </text>
-        </>
+
+            <ElevatorAxis side={side} x={0} y={5} />
+
+            <HydraulicIndicator x={side === 'left' ? -60 : 18} y={79} type={leftHydraulicSystem} />
+            <HydraulicIndicator x={side === 'left' ? -38 : 40} y={79} type={rightHydraulicSystem} />
+        </SvgGroup>
     );
 };
 
+const ElevatorAxis = ({ x, y, side }: ComponentPositionProps & ComponentSidePositionProps) => {
+    const d1 = `M0 0 l${
```

Comment:
Is there a reason for naming the constant `d1`? It’s not entirely clear to me as the content string doesn’t disclose the meaning when reading the code. Same applies for the other appearances of d1, d2 etc.

Reply:
I've not adapted the existing names in this case. I'm guessing `d` is used as that is the HTML attribute on the `path` element. 
Renaming these would likely result in names such as `lowerHorizontalPartOfWing`, `verticalStripeTowardsArrow` or who knows what other crufty name. I can do it, but if this is the only concern I'd rather avoid starting up the whole development workflow just for this 😀.

Reply:
Agreed :D